src/Command/Environment/EnvironmentXdebugCommand.php

```diff
@@ -0,0 +1,108 @@
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->validateInput($input);
+        $this->getSelectedEnvironment();
+
+        $container = $this->selectRemoteContainer($input);
+        $sshUrl = $container->getSshUrl();
+
+        $config = $container->getConfig()->getNormalized();
+        $key = isset($config['runtime']['xdebug']['key']) ? $config['runtime']['xdebug']['key'] : '';
+
+        if (!$key) {
+            $output->getErrorOutput()->writeln(
+                "<error>A debugging key has not been found</error>\n" .
+                "\n" .
+                "To use Xdebug your project must have a <info>debugging key</info> informed.\n" .
+                "Such key is informed in the <info>.platform.app.yaml</info> file as in this example:\n" .
+                "\n" .
+                "<info>...\n" .
+                "runtime:\n" .
+                "    xdebug:\n" .
+                "        key: <options=underscore>secret_key</>"
+            );
+
+            return null;
+        }
+
+
+        /** @var \Platformsh\Cli\Service\Ssh $ssh */
+        $ssh = $this->getService('ssh');
+
+        // The socket is removed to prevent 'file already exists' errors
+        $commandCleanup = $ssh->getSshCommand();
+        $commandCleanup .= ' ' . escapeshellarg($sshUrl) . ' rm -f ' . escapeshellarg(self::SOCKET_PATH);
+        $this->debug("Cleanup command: " . $commandCleanup);
+        $process = new Process($commandCleanup);
+        $process->run();
+
+        $output->writeln("Starting the tunnel for Xdebug.");
+
+        // Set up the tunnel
+        $port = $input->getOption('port');
+
+        $sshOptions = [];
+        $sshOptions['ExitOnForwardFailure'] = 'yes';
+
+        $commandTunnel = $ssh->getSshCommand($sshOptions) . ' -TNR ' . escapeshellarg(self::SOCKET_PATH . ':127.0.0.1:' . $port);
+        $commandTunnel .= ' ' . escapeshellarg($sshUrl);
+        $this->debug("Tunnel command: " . $commandTunnel);
+        $process = new Process($commandTunnel);
+        $process->setTimeout(null);
+        $process->start();
+
+        usleep(100000);
+
+        if (!$process->isRunning() && !$process->isSuccessful()) {
+            $this->stdErr->writeln(trim($process->getErrorOutput()));
+            $this->stdErr->writeln('Failed to create the tunnel.');
+            return $process;
+        }
+
+        $output->writeln(
+            sprintf(
+                "\nThe Xdebug tunnel is set up. To break it, close this command by pressing <info>CTRL+C</info>.\n " .
+                "\n" .
+                "To debug, you must either set a cookie like '<info>XDEBUG_SESSION=%s</info>' or append '<info>XDEBUG_SESSION_START=%s</info>' as a query string when visiting your project.",
+                $key, $key
+            )
+        );
+
+        $process->wait();
+
+        return $process;
```

Comment:
return should be an integer

```suggestion
        return $process->wait();
```